src/Tribe/Repositories/Attendee/RSVP.php

```diff
@@ -104,4 +105,126 @@ public function checked_in_keys() {
 		return $this->limit_list( $this->key_name, parent::checked_in_keys() );
 	}
 
+	/**
+	 * {@inheritDoc}
+	 */
+	public function update_additional_data( $attendee, $ticket, $attendee_data ) {
+
+		$attendee_data = $this->format_attendee_data( $attendee, $ticket, $attendee_data );
+
+		try {
+			$query = $this->where( 'id', $attendee->ID )
+			              ->set( 'ticket_id', $ticket->ID )
+						  ->set( 'full_name', $attendee_data['full_name'] )
+						  ->set( 'email', $attendee_data['email'] )
+						  ->set( 'event_id', $attendee_data['event_id'] )
+						  ->set( 'security_code', $attendee_data['security_code'] )
+						  ->set( 'order_id', $attendee_data['order_id'] )
+						  ->set( 'optout', $attendee_data['optout'] )
+						  ->set( 'attendee_status', $attendee_data['attendee_status'] )
+						  ->set( 'price_paid', $attendee_data['price_paid'] )
+						  ->set( 'user_id', $attendee_data['user_id'] );
+		}
+		catch ( Tribe__Repository__Usage_Error $e ) {
+			do_action( 'tribe_log', 'error', __CLASS__, [
```

Comment:
* :no_entry_sign: Opening parenthesis of a multi-line function call must be the last content on the line
* :no_entry_sign: Only one argument is allowed per line in a multi-line function call
* :no_entry_sign: Only one argument is allowed per line in a multi-line function call
* :no_entry_sign: Only one argument is allowed per line in a multi-line function call

<sub>hash: 626874f2d2217517a9f92cb75e665a41</sub>